src/modules/share/components/material_ui/stepper/VerticalLinearStepper.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState } from 'react';
+// material
+import { Box, Step, Paper, Button, Stepper, StepLabel, Typography, Grid } from '@mui/material';
+import VerifyForm from 'src/modules/sessions/components/VerifyForm';
+// ----------------------------------------------------------------------
+
+const steps = ['Your information', 'Secret question', 'Status'];
```

Comment:
please make this component to be extendable, allow passing steps and component props